goss-testing/scripts/data-validator.py

```diff
@@ -192,103 +206,98 @@ def are_hosts_sane(data, desired_keys):
 
 
 def validate_ntp(data):
```

Comment:
It's just me, but since we're using Python3 we can use annotations.

```python
def validate_ntp(data):
```

vs.

```python
# define input types and return type
def validate_ntp(data: dict) -> int
```

So this is a nit, but if you wanted to annotate it does help maintainers see/read what's happening.

Common return types:
```python
def foo(bar: list) -> None
def foo(**kwargs) -> Bool
```

Reply:
I wouldn't worry about implementing this for today, necessarily. The most urgent thing is to fix up the test to avoid the current false failures, before we start seeing them on live systems.

Reply:
I'll save this for later.